libs/src/services/creatorNode/index.js

```diff
@@ -17,6 +17,28 @@ class CreatorNode {
    */
   static getSecondaries (endpoints) { return endpoints ? endpoints.split(',').slice(1) : [] }
 
+  /**
+   * Checks if a download is available from provided creator node endpoints
+   * @param {string} endpoints creator node endpoints
+   * @param {number} trackId
+   */
+  static async checkIfDownloadAvailable (endpoints, trackId) {
+    for await (const endpoint of endpoints.split(',')) {
```

Comment:
assuming `endpoints` refers to on-chain user `creator_node_endpoints` field in [primary, ...secondaries] order, is there ever any value in checking download_status on secondaries? no way for secondaries to return cid if primary doesn't

Reply:
@raymondjacobson